rust/repositories.bzl

```diff
@@ -639,6 +651,12 @@ _RUST_ANALYZER_TOOLCHAIN_TOOLS_REPOSITORY_ATTRS = {
             "See [repository_ctx.download](https://docs.bazel.build/versions/main/skylark/lib/repository_ctx.html#download) for more details."
         ),
     ),
+    "netrc": attr.string(
```

Comment:
I think buildifier is failing because these keys aren't sorted?

Reply:
Fixed.
BTW, somehow I didn't manage to make buildifier to complain about it locally. Is there some specific configuration or something like that?